### PR TITLE
Add fetch.js

### PR DESCRIPTION
**Pull request by lukwys**

### CODE REVIEW

File: src/API/fetch.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+function loadDoc(){
```
Comment (benedyktdryl) [inline src/API/fetch.js]: Great start. Let's use `fetch` API:
https://davidwalsh.name/fetch

Comment (benedyktdryl) [inline src/API/fetch.js]: Be more descriptive about naming.

File: Readme.MD [hunk 2]
```diff
@@ -40,6 +40,9 @@ This application is showing statistics from several football stats APIs.
 - Lukasz Wysocki <lukwys>
 - Damian Rutkowski <Txwxcc>
 
+###API
```
Comment (benedyktdryl) [inline Readme.MD]: Space between `###` and API

File: Readme.MD [hunk 3]
```diff
@@ -40,6 +40,9 @@ This application is showing statistics from several football stats APIs.
...
+- football-data.org `63e07dbf62ac463d9aa94a83443512ec`
```
Comment (benedyktdryl) [inline Readme.MD]: What is this hash?

Comment (Txwxcc) [inline Readme.MD]: This is the api-key for football-data.org

**Comment by Txwxcc** [comment]
Why name of directory `api/` is upper-cased with no reason?
Can you change it before merge?

**Pull request closed without merging**